lib/user.ts

```diff
@@ -6,9 +10,14 @@ const displayNameElement = document
 	.getElementsByTagName('head')[0]
 const displayName = displayNameElement ? displayNameElement.getAttribute('data-user-displayname') : null
 
+const isAdmin = (typeof OC === 'undefined')
+	? false
+	: OC.isUserAdmin()
+
 export interface NextcloudUser {
```

Comment:
Should we also just extend this from `Nextcloud.Common.CurrentUser`?

Reply:
Sure!

Reply:
Actually nevermind. `Nextcloud.Common.CurrentUser` allows `uid` to be false, which will never be the case here.

Reply:
Right, there was something :stuck_out_tongue_closed_eyes: